discover/commerce/articles/01-catalog-management/03-options.markdown

```diff
@@ -0,0 +1,147 @@
+# Options [](id=options)
+
+**An option** is a product feature that can be selected or specified by the
+buyer.  An option could prompt a buyer to choose between engines of different
+displacements, building materials of different dimensions, or T-shirts of
+different colors.
+
+To create an option, you must first create an option template and then a set of
+option values. You must also 
+[assign the template to a product](/web/liferay-emporio/documentation/-/knowledge_base/7-1/assigning-options-to-products).
+
+**An Option Template** describes the kind of variation that the option defines.
+If the buyer will be prompted to choose between product variants of different
+sizes, then the template would be defined as *Size*.
+
+**An Option Value** refers to a specific selection a buyer might choose. For
+example, an option template called *Size* might be assigned the option values
+*Small*, *Medium*, and *Large*.
+
++$$$
+
+Note: not every option requires option values. Values must be defined only when
+buyers will be prompted to choose from among preset variants. Values are not
+required for options that are more open-ended: for instance, the buyer might be
+prompted to enter custom text to be printed or engraved on the product, or asked
+to provide schematics of a custom product that will be fabricated or 3D-printed.
+
+You can select whether an option template requires values when you set its field
+type (see below).
+
+$$$
+
+**Scope:** When an option template is created, it is scoped to the catalog. This
+means that the template's configuration is applied to every product to which it
+is assigned. However, when you deploy a template to an individual product,
```

Comment:
Reword to avoid repetition of "is created...is scoped...is applied...is assigned." Perhaps this: 

When you create an option template, its default scope is the catalog. Every product it's assigned to inherits the template's configuration.